spec/index.html

```diff
@@ -587,8 +587,9 @@ <h3>IRIs</h3>
       from [[RFC3987]] is provided in <a href="#iri-abnf" class="sectionRef"></a>.</p>
 
     <p>IRIs in the RDF abstract syntax MUST be <a data-cite="RFC3986#section-5">resolved</a>
-      per [[RFC3986]],
-      and MAY contain a fragment identifier.</p>
+      per [[RFC3986]], and MAY contain a fragment identifier. IRIs SHOULD
```

Comment:
As suggested in [my review](https://lists.w3.org/Archives/Public/public-rdf-star-wg/2025Jan/0111.html),
I would suggest to rephrase this as follows
(sorry, I can not use the suggestion mode, because the first line is not part of the diff)

> IRIs in the RDF abstract syntax MAY NOT be relative references, and MUST be resolved per [RFC3986]. They MAY contain a fragment identifier.

I like the addition about following the rules defined by the scheme.

Reply:
Added. MY NOT is not in RFC 2119. I think you meant "MUST NOT" strength condition.